proposals/lambda-discard-parameters.md

```diff
@@ -0,0 +1,21 @@
+# Lambda discard parameters
+
+## Summary
+
+Allow discards (`_`) to be used as lambda parameters.
+For example:
+- `(_, _) => 0`
+- `(int _, int _) => 0`
+- `delegate(int _, int _) { return 0; }`
+
+## Motivation
+
+Unused lambda parameters do not need to be named. The intent of discards is clear, i.e. they are unused/discarded.
+
+## Detailed design
+
+In a lambda or delegate declaration with more than one parameter named `_`, those parameters are discard parameters.
```

Comment:
>delegate declaration [](start = 15, length = 20)

Did you really mean "delegate declaration", that is, a declaration of a delegate type?